### DISCUSSION TITLE
Utilize unique tiering (weighting) data from poorfishwife

### DISCUSSION THREAD

**Discussion by cdrg** [xanthics/poe_gen_gwennen#18] [Ideas]
Using poorfishwife's unique data, it should be possible (if probably not trivial) to have the option to sort / exclude bases depending on the likelihood of obtaining the valuable result, or in other words the chaos value per gamble.

A complication of course is that Gwennen uses a modified drop table, but I think that's mentioned in there somewhere.

https://www.reddit.com/r/pathofexile/comments/sjnlwq/anglers_mate_10_the_droprate_of_reefbane_and_the/

**Comment by xanthics** [comment]
While Gwennen's table is mentioned, that list of rarity is from 3.14 so it is missing everything since then.  GGG can, and does, modify drop tables as well so there may be outdated information in that data set.  I'm not completely opposed to adding the data that said.  However there are enough caveats at the moment to make me question the value.
